bitnami/redis/templates/_helpers.tpl

```diff
@@ -1,182 +1,84 @@
 {{/* vim: set filetype=mustache: */}}
```

Comment:
I found these two helper references in the yaml 

```
helper redis.dhParams used in the yaml is not defined
helper redis.tplValue used in the yaml is not defined
```

Check them and fix (I cannot put the exact place because of the vscode plugin)